src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -277,8 +307,8 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
-    fluid.slider.combineSliderOptions = function (sliderOptions, model, range) {
-        return $.extend(true, {}, sliderOptions, model, range);
+    fluid.slider.combineSliderOptions = function (sliderOptions, model, range, ariaOptions, strings) {
+        return $.extend(true, {}, sliderOptions, model, range, ariaOptions, strings);
```

Comment:
I suspect it's safest to not do this unnecessarily for those reasons, so I'll split them out again.